### PR TITLE
Test batching with jest

### PR DESCRIPTION
**Pull request by jesstelford**
For my own peace of mind, I wanted to ensure the batching was working correctly. I've added a single Jest test to replace the existing test (which didn't seem to work anyway), exercising the batching logic.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
+/*
```
Comment (jesstelford) [inline jest.config.js:1]: Jest auto-generated this file.

Comment (Sinequanonh) [reply jest.config.js:1]: Sweeeet, thanks a lot, that's awesome!!
I'll have a look and get it merged

**Comment by Sinequanonh** [comment]
Sweeeet, thanks a lot, that's awesome!!

**Comment by jesstelford** [comment]
@Sinequanonh Anything else needed to get this merged?

**Comment by jesstelford** [comment]
🎉 Thanks!

**Pull request merged**